converter/har/converter.go

```diff
@@ -55,7 +55,9 @@ func Convert(h HAR, includeCodeCheck bool, returnOnFailedCheck bool, batchTime u
 		fmt.Fprintf(w, "// %v\n", h.Log.Comment)
 	}
 
-	fmt.Fprint(w, "\n")
+	// recordings include redirections as separate requests, and we dont want to trigger them twice
+	fmt.Fprint(w, "\nexport let options = { maxRedirects: 0 };\n\n")
```

Comment:
Is it always the case with HAR files that redirects are included as separate requests? If not then we should perhaps make this a CLI flag as well.

Reply:
The HAR spec doesnt mention anything about redirects (except as part of responses) which would indicate that redirects are always represented as a separate request, but I'll double check using a browser based recorder (my proxy based recorder did generate separate requests)

Reply:
Confirmed. See attached file if you dont believe me ;)



[www.google.se.har.json.txt](https://github.com/loadimpact/k6/files/1699736/www.google.se.har.json.txt)

Reply:
Alright, cool, I believe you. Just checking :)

Reply:
:) not a bad question, I hadnt really double checked..